main/python/rootcp.py

```diff
@@ -49,10 +55,26 @@ def execute():
 	parser = get_argparse()
 	 # Put arguments in shape
 	sourceList, destFileName, destPathSplit, optDict = cmdLineUtils.getSourceDestListOptDict(parser)
-
+	try:
+		algorithm = optDict["compressalglvl"][0]
+	except (TypeError, IndexError):
+		algorithm = None
+	try:
```

Comment:
This should probably only be run if the previous one succeeded.

Reply:
Agree, but no longer relevant given my comment in https://github.com/root-project/root/pull/14714#issuecomment-1952406831